hydra/_internal/core_plugins/importlib_resources_config_source.py

```diff
@@ -40,23 +41,32 @@ def load_config(
         if not res.exists():
             raise ConfigLoadError(f"Config not found : {normalized_config_path}")
 
-        with res.open(encoding="utf-8") as f:
-            header_text = f.read(512)
+        zipfile_path = sys.version_info[0:2] >= (3, 8) and isinstance(res, zipfile.Path)
+        try:
+            if zipfile_path:
```

Comment:
```suggestion
        try:
            if sys.version_info[0:2] >= (3, 8) and isinstance(res, zipfile.Path):
```

Reply:
Maybe, your call :)

Reply:
fixed, sorry shoulda spotted that myself :)